refinery/core/serializers.py

```diff
@@ -82,6 +82,31 @@ def partial_update(self, instance, validated_data):
         return instance
 
 
+class SiteProfileSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SiteProfile
+        fields = ('about_markdown', 'site', 'intro_markdown',
+                  'twitter_username',
+                  'yt_videos')
+
+    def partial_update(self, instance, validated_data):
+        """
+        Update and return an existing `DataSet` instance, given the
+        validated data.
+        """
+        instance.about_markdown = validated_data.get('about_markdown',
+                                                     instance.about_markdown)
+        instance.intro_markdown = validated_data.get('intro_markdown',
+                                                     instance.intro_markdown)
+        instance.twitter_username = validated_data.get(
+            'twitter_username',  instance.twitter_username
+        )
+        instance.yt_videos = validated_data.get('yt_videos',
+                                                instance.yt_videos)
+        instance.save()
+        return instance
```

Comment:
I think the shorthand for this would look like:  `instance.update(**validated_data)`